trunion/tests/base.py

```diff
@@ -0,0 +1,82 @@
+# ***** BEGIN LICENSE BLOCK *****
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+# ***** END LICENSE BLOCK *****
+
+import os
+
+from pyramid import testing
+from mozsvc.config import load_into_settings
+from mozsvc.tests.support import TestCase
+import trunion.crypto as crypto
+
+# Needed to inspect signatures
+from base64 import b64decode
```

Comment:
normally stdlib imports go up at the top, below `import os` the import rules we normally follow are http://mozweb.readthedocs.org/en/latest/reference/python-style.html#import-statements which is basically the pep8 import order